app/src/main/res/drawable/ic_delete_white_24dp.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+  ~ Nextcloud Talk application
+  ~
+  ~ @author Mario Danic
+  ~ Copyright (C) 2017-2018 Mario Danic <mario@lovelyhq.com>
```

Comment:
That's not you ;-)
I suspect that there is still a wrong template somewhere?

Reply:
I'll keep it that way because I "only" copied the black symbol and changed it to white.